tox.ini

```diff
@@ -4,17 +4,23 @@
 # When updating the envlist, be sure to also update TOX_ENV in .travis.yml
 envlist =
     docs,
-    {py27,py34,py35}-django1.8-drf{3.2,3.3}
-    # Django 1.9 requires DRF >= 3.3
-    {py27,py34,py35}-django1.9-drf3.3
+    py{27,34,35}-django{1.8,1.9,1.10}-drf{3.4,3.5},
+    # Django 1.11+ is only supported as of django-rest-framework 3.5.
+    # Python 3.6 is only supported as of Django 1.11.
```

Comment:
Yeah this comment says "Python 3.6 is only supported as of Django 1.11", whereas I think you might have read it as "Django 1.11 only supports Python 3.6".